### ISSUE TITLE
Add option to fine tune (load from pre-trained state) and modify training parameters

### ISSUE THREAD

**Issue by marjanfamili** [alan-turing-institute/affinity-vae#27]
Implement re-start training from existing state (fine tuning flag?).

Also would be nice to be able to change the training parameters for example learning rate.
